{"url":"https://api.github.com/repos/willmcgugan/rich/issues/215","repository_url":"https://api.github.com/repos/willmcgugan/rich","labels_url":"https://api.github.com/repos/willmcgugan/rich/issues/215/labels{/name}","comments_url":"https://api.github.com/repos/willmcgugan/rich/issues/215/comments","events_url":"https://api.github.com/repos/willmcgugan/rich/issues/215/events","html_url":"https://github.com/willmcgugan/rich/issues/215","id":674269636,"node_id":"MDU6SXNzdWU2NzQyNjk2MzY=","number":215,"title":"Error with Python 3.6.0 [BUG]","user":{"login":"resparis","id":49069898,"node_id":"MDQ6VXNlcjQ5MDY5ODk4","avatar_url":"https://avatars1.githubusercontent.com/u/49069898?v=4","gravatar_id":"","url":"https://api.github.com/users/resparis","html_url":"https://github.com/resparis","followers_url":"https://api.github.com/users/resparis/followers","following_url":"https://api.github.com/users/resparis/following{/other_user}","gists_url":"https://api.github.com/users/resparis/gists{/gist_id}","starred_url":"https://api.github.com/users/resparis/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/resparis/subscriptions","organizations_url":"https://api.github.com/users/resparis/orgs","repos_url":"https://api.github.com/users/resparis/repos","events_url":"https://api.github.com/users/resparis/events{/privacy}","received_events_url":"https://api.github.com/users/resparis/received_events","type":"User","site_admin":false},"labels":[{"id":1668492285,"node_id":"MDU6TGFiZWwxNjY4NDkyMjg1","url":"https://api.github.com/repos/willmcgugan/rich/labels/wontfix","name":"wontfix","color":"ffffff","default":true,"description":"This will not be worked on"}],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2020-08-06T12:19:58Z","updated_at":"2020-08-06T14:56:18Z","closed_at":"2020-08-06T14:56:18Z","author_association":"NONE","active_lock_reason":null,"body":"Using Python 3.6.0 from Anaconda, I am getting the following error when I tried to import it.\r\n\r\n>>> console = Console()\r\nTraceback (most recent call last):\r\n  File \"<stdin>\", line 1, in <module>\r\nNameError: name 'Console' is not defined\r\n>>> from rich.console import Console\r\nTraceback (most recent call last):\r\n  File \"<stdin>\", line 1, in <module>\r\n  File \"\\project\\rich\\lib\\site-packages\\rich\\console.py\", line 30, in <module>\r\n    from . import errors, themes\r\n  File \"project\\rich\\lib\\site-packages\\rich\\themes.py\", line 1, in <module>\r\n    from .default_styles import DEFAULT_STYLES\r\n  File \"\\project\\rich\\lib\\site-packages\\rich\\default_styles.py\", line 18, in <module>\r\n    strike=False,\r\n  File \"\\project\\rich\\lib\\site-packages\\rich\\style.py\", line 115, in __init__\r\n    self._color = None if color is None else _make_color(color)\r\n  File \"\\project\\rich\\lib\\site-packages\\rich\\style.py\", line 113, in _make_color\r\n    return color if isinstance(color, Color) else Color.parse(color)\r\nAttributeError: type object 'Color' has no attribute 'parse'","closed_by":{"login":"willmcgugan","id":554369,"node_id":"MDQ6VXNlcjU1NDM2OQ==","avatar_url":"https://avatars3.githubusercontent.com/u/554369?v=4","gravatar_id":"","url":"https://api.github.com/users/willmcgugan","html_url":"https://github.com/willmcgugan","followers_url":"https://api.github.com/users/willmcgugan/followers","following_url":"https://api.github.com/users/willmcgugan/following{/other_user}","gists_url":"https://api.github.com/users/willmcgugan/gists{/gist_id}","starred_url":"https://api.github.com/users/willmcgugan/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/willmcgugan/subscriptions","organizations_url":"https://api.github.com/users/willmcgugan/orgs","repos_url":"https://api.github.com/users/willmcgugan/repos","events_url":"https://api.github.com/users/willmcgugan/events{/privacy}","received_events_url":"https://api.github.com/users/willmcgugan/received_events","type":"User","site_admin":false},"performed_via_github_app":null}